app/main.py

```diff
@@ -1,6 +1,20 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str) -> None:
+    hall_number = CinemaHall(number=hall_number)
+    cleaner = Cleaner(name=cleaner)
+    cb = CinemaBar
```

Comment:
```suggestion
    cb = CinemaBar
```
the extra line is extra